.github/workflows/build.yml

```diff
@@ -169,7 +169,7 @@ jobs:
           cp $GITHUB_WORKSPACE/LICENSE "$LICENSE_DIR/LICENSE.txt"
 
       - name: Package rpm and deb files
-        if: matrix.goos == 'linux' && matrix.component == 'cli' && matrix.skip_packaging != 'true'
+        if: matrix.goos == 'linux' && matrix.component != 'control-plane' && matrix.skip_packaging != 'true'
```

Comment:
~ Not a big deal, but here and below, I'd prefer we were explicit rather than inverting the match, in case other components are added in the future:
```yaml
contains(fromJSON('["cli", "control-plane/cni"]'), matrix.component)
```

That said, I don't think we need both checks at this point. I think the ideal thing might be to change `skip_packaging` to `package_linux` (opt-in), and set it for the matrix combos we want to package - or add `skip_packaging` to the `linux` `control-plane` values. Either way, at that point we could just remove the name match entirely, and have a clear value in the matrix that says whether a linux build is packaged.

@shore @david-yu WDYT?

Reply:
I can just add skip packaging manually to everything that needs to be skipped that might be easier.